purview/models/product.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from neomodel import UniqueIdProperty, RelationshipFrom, StringProperty
+
+from purview.models.base import PurviewStructuredNode
+
+
+class Product(PurviewStructuredNode):
+    """Definition of a released Product"""
+
+    attached_koji_tag = RelationshipFrom('.koji.KojiTag', 'ATTACHED_TO')
+    attached_advisory = RelationshipFrom('errata.Advisory', 'ATTACHED')
+    id_ = UniqueIdProperty(db_property='id')
```

Comment:
Is this unique ID coming from the Errata Tool? If so, we should consider moving this class to `errata.py`.

Reply:
yes, but I made a separate file so that there is no confusion with the class. But sure, I can move it